scripts/03-deploy-dfxcad-cadc-pool-univ3.js

```diff
@@ -0,0 +1,136 @@
+import IUniswapV3FactoryArtifact from '@uniswap/v3-core/artifacts/contracts/interfaces/IUniswapV3Factory.sol/IUniswapV3Factory.json';
+import IUniswapV3PoolArtifact from '@uniswap/v3-core/artifacts/contracts/interfaces/IUniswapV3Pool.sol/IUniswapV3Pool.json';
+
+import DfxCadLogicV1Artifact from '../out/DfxCadLogicV1.sol/DfxCadLogicV1.json';
+import MUNILogicV1Artifact from '../out/MUNILogicV1.sol/MUNILogicV1.json';
+import ASCUpgradableProxyArtifact from '../out/ASCUpgradableProxy.sol/ASCUpgradableProxy.json';
+
+import { encodeSqrtRatioX96, FeeAmount, TickMath } from '@uniswap/v3-sdk';
+
+const { formatUnits } = ethers.utils;
+
+// Tokens
+const CADC = "0xcaDC0acd4B445166f12d2C07EAc6E2544FbE2Eef";
+const DFXCAD = "0xFE32747d0251BA92bcb80b6D16C8257eCF25AB1C";
+
+// Uniswap contracts
+const UNISWAP_FACTORY = "0x1F98431c8aD98523631AE4a59f267346ea31F984";
+
+// Access controller
+const DFX_GOV_MULTISIG = '0x27E843260c71443b4CC8cB6bF226C3f77b9695AF'
+
+
+const main = async () => {
+    const dfxCadToken = new ethers.Contract(DFXCAD, DfxCadLogicV1Artifact.abi, wallet);
+    const cadcToken = new ethers.Contract(CADC, FiatTokenV2Artifact, wallet);
+    const uniswapV3Factory = new ethers.Contract(UNISWAP_FACTORY, IUniswapV3FactoryArtifact.abi, wallet);
+    const MUNILogicFactory = new ethers.ContractFactory(
+        MUNILogicV1Artifact.abi, MUNILogicV1Artifact.bytecode.object, wallet
+    );
+    const UpgradableProxyFactory = new ethers.ContractFactory(
+        ASCUpgradableProxyArtifact.abi, ASCUpgradableProxyArtifact.bytecode.object, wallet
+    );   
+
+    // Check token addresses both exist
+    const dfxCadBalance = await dfxCadToken.balanceOf(wallet.address);
+    const cadcBalance = await cadcToken.balanceOf(wallet.address);
+    console.log(`Total balance--dfxCad: ${formatUnits(dfxCadBalance)} cadc: ${formatUnits(cadcBalance)}`);
+
+    // Create the new Uniswap pool
+    console.log("Creating UniswapV3 dfxCAD/CADC pool...");
+    const feeAmount = FeeAmount.LOWEST;
+    await uniswapV3Factory.createPool(dfxCadToken.address, cadcToken.address, feeAmount);
+
+    // Fetch pool's address
+    const poolAddress = await uniswapV3Factory.getPool(dfxCadToken.address, cadcToken.address, feeAmount);
+    console.log("Pool address:", poolAddress)
+
+    // Instantiate ethers ABI on pool address
+    console.log("Initializing UniswapV3 dfxCAD/CADC pool at 1 dfxCAD:1CADC...")
+    const DfxCadCadcPool = new ethers.Contract(
+        poolAddress, IUniswapV3PoolArtifact.abi, wallet,
+    );
+
+    // Create sqrtPrice from 1 dfxCAD:1 CADC and initialize pool
+    const sqrtRatioX96 = encodeSqrtRatioX96(1, 1);
+    await DfxCadCadcPool.connect(wallet).initialize(sqrtRatioX96.toString());
+
+    // Increase TWAP window like in Muni.t.sol tests
+    await DfxCadCadcPool.increaseObservationCardinalityNext(5);
+
+    const tickSpacing = await DfxCadCadcPool.tickSpacing();
+    const lowerSqrtPriceX96 = encodeSqrtRatioX96(1000, 1004);
```

Comment:
I would give it a little bit more variance, like maybe 0.95 -> 1.05 just in case

What do you guys think?

Reply:
Both seem fine to me, just curious your reasoning for this

Reply:
I guess we could do 0.98 <-> 1.02 instead

My biggest concern is that we're artificially creating an extreme wall at 1.00 which doesn't encourage arbitrage